web/client/components/mapcontrols/search/SearchBar.jsx

```diff
@@ -115,6 +115,8 @@ export default ({
     }, [searchOptions?.services]);
 
     useEffect(() => {
+        // clear coordinate marker in case of change map CRS
+        onClearCoordinatesSearch({owner: "search"});
```

Comment:
Why is it needed to clear the marker, since the value is reprojected to CRS so the marker should be at the same position, right?